UPGRADE-6.0.md

```diff
@@ -17,6 +17,7 @@
 - Removed `Mautic\CoreBundle\Factory\MauticFactory::getDebugMode` use dependency injection instead.
 - Removed `Mautic\CoreBundle\Factory\MauticFactory::getMauticBundles` use BundleHelper instead.
 - Removed `Mautic\CoreBundle\Factory\MauticFactory::getKernel` use dependency injection instead.
+- Removed `Mautic\CoreBundle\Factory\MauticFactory::getVersion` use dependency injection with KernelInterface, which will retrieve \AppKernel, then invoke getVersion() method.
```

Comment:
I found that there is the https://github.com/mautic/mautic/blob/6.x/app/bundles/CoreBundle/Helper/AppVersion.php service that might be better suited for getting the version. Although it does the same thing as what you suggest.
```suggestion
- Removed `Mautic\CoreBundle\Factory\MauticFactory::getVersion()` use dependency injection with Mautic\CoreBundle\Helper\AppVersion::getVersion() instead.
```

Reply:
@escopecz  This is *VERY* debatable, since AppVersion is not mockable, because of a static method. And in the end it has no difference, if i just use the constant directly, or AppVersion: both are static, both can not be mocked.

Reply:
Fair point 👍